packages/shared/examples/shaped-scatter-plot/data.ts

```diff
@@ -0,0 +1,107 @@
+export type DataRecord = {
+  name: string;
+  owner: string;
+  trainedParam: number;
+  date: string;
+};
+
+
+export const categories = [
+  'Amazon',
+  'Anthropic',
+  'OpenAI',
+  'Google',
+  'Meta',
+  'Other',
+]
+
+export const shapes = [
+  'circle',
+  'cross',
+  'diamond',
+  'square',
+  'star',
+  'triangle',
+]
+
+export const sumCategories = (category: string): string => {
```

Comment:
I'm not sure this function is necessary. I see you're using it to map category to the color/shape, but `scaleOrdinal` should already implement that logic for you. See my other comment below

Reply:
`sumCategories` also put any owner that's not in `const categories` into `Other`.

Reply:
Fair point. I think it still works without it but this way it clearer, so let's keep it

Reply:
Do you mean we can get rid of `sumCategories` entirely and call `getColor = (d: DataRecord): string => colorScale(d.owner)` and `shapeScale(d.owner)` directly? I'm not sure if that would work. Because then, in the data set, if the `owner` isn't part of the 6 categories, (like owner: 'Inflection'), it will get mapped to a different shape and color instead of `Other` right? Or am I missing something? 

I don't want to have inaccurate examples, so I'm happy to update if that's the case!

Reply:
@lee00678 Yeah so I actually just tried it myself and it does work (feel free to try it too and verify). I think under the hood the `scaleOrdinal` automatically bins extraneous values into the last value of the provided range.

But that doesn't mean your example is inaccurate, if anything it is more clear. So I approved this PR you can go ahead and merge if you like. Or if you want to change it, that's fine too. I'll leave it up to you :)

Reply:
Thanks for the review! I didn't know this feature about `scaleOrdinal`.